app/components/login/login.component.ts

```diff
@@ -20,11 +23,13 @@ export class LoginComponent {
   serverColor: string;
   usernameColor: string;
   passwordColor: string;
+  private _translations: typeof Translations = Translations;
 
   constructor(private fb: FormBuilder,
               private auth: SessionAuthenticationService,
               private session: SessionService,
```

Comment:
I think using names like `_formBuilder`, `_authenticationService`, `_sessionService`, would make for more readable code. While reading code without context, it's hard to guess what something like `fb` might be.